src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/imports/illegalimport/InputIllegalImportDefault.java

```diff
@@ -1,7 +1,7 @@
-////////////////////////////////////////////////////////////////////////////////
-// Test case file for checkstyle.
-// Created: 2001
-////////////////////////////////////////////////////////////////////////////////
+//non-compiled with javac: contains specially crafted set of imports for testing
```

Comment:
can we keep it to be compiled by jdk8 javac ?

we are not going to drop java8 completely in near future, but it is completely ok to update all sources to be compiled by most rescent jdk, but Travis should still try to compile old files with old jdk, just to keep compilation status of file.

Reply:
done